app/controllers/twilio_webhooks_controller.rb

```diff
@@ -54,8 +54,34 @@ def create_incoming_text_message
     head :ok
   end
 
+  def outbound_call_connect
+    @outbound_call = OutboundCall.find(params[:id])
+    twiml = Twilio::TwiML::VoiceResponse.new
+    # The status callback for the call is attached to the dial event to the client.
+    # This means that the length of the call will be based on how long the user was connected to the client,
+    # And the status will be based on whether the client picked up the call.
+    twiml.dial do |dial|
+      dial.number(@outbound_call.to_phone_number,
+                  status_callback_event: 'answered completed',
+                  status_callback: webhook_url,
+                  status_callback_method: 'POST')
+    end
+    render xml: twiml.to_xml
+  end
+
   private
 
+  def webhook_url
```

Comment:
Webhook_url is too general now that its in the webhooks controller! update name.